src/index.js

```diff
@@ -163,28 +169,35 @@ OAuth.prototype.refresh = function (url, callback) {
  *
  * @param  {String}   url      The token endpoint URL
  * @param  {String}   clientId The client id
+ * @param  {String}   scopes The scopes with oauth
  * @param  {Function} callback First arg is error (if any), second is OAuth object on success.
  */
-OAuth.authorizeImplicitly = function(url, clientId, callback) {
-	var self = this,
+OAuth.authorizeImplicitly = function (url, clientId, scopes, callback) {
```

Comment:
Adding a new parameter to all these methods breaks existing usage currently. There's no way to not provide scopes but provide a callback function (which is required). You need to add some better arg sniffing to check if "scopes" is actually the callback function the user passed because they opted not to pass in scopes explicitly (so basically your `var scope = scopes || ''` is useless unless they drop the scopes and callback args).

Maybe something more like:
```javascript
if (typeof scopes == 'function') {
  callback = scopes;
  scopes = '';
}
```
And drop the extra `scope` variable altogether in the body and just refer to `scopes`